pkg/commands/flagsets.go

```diff
@@ -62,6 +62,8 @@ func setupRunFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
 func setupOutputFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
 	internal.AddFlagAndBind(v, fs, fs.String, "path-prefix", "output.path-prefix", "",
 		color.GreenString("Path prefix to add to output"))
+	internal.AddFlagAndBind(v, fs, fs.String, "path-mode", "output.path-mode", "",
+		color.GreenString("Path mode to use"))
```

Comment:
I propose extending descriptions for `path-prefix` and `path-mode` with possible values.

It's for consistency with the rest of descriptions.
<img width="1133" alt="image" src="https://github.com/user-attachments/assets/b23960f9-9831-4142-a5b8-c988a7a34322" />

Reply:
There are no "possible values" `path-prefix`